templates/pages/index.hbs

```diff
@@ -31,6 +31,18 @@
       {{/is}}
     {{/eachReverse}}
   </div>
+</section> --}}
+
+<section class="upcoming">
+  <article class="next-meeting" style="padding: 2em 3em">
+    <p>
+      We appreciate everyone coming to Gem City JS over the years. We have decided to take a hiatus for a bit.
+      We will send out an email and tweet when we decide to come back.
```

Comment:
Could we suggest some other excellent meetups they might attend with a link to the lamppost site?

We appreciate everyone coming to Gem City JS over the years. We have decided to take a hiatus for a bit.
We will send out an email and tweet when we decide to come back.
[Check out one of the other excellent web meetups in the area](https://lamppost.io/).